app/services/decisions/evidence_decision_tree.rb

```diff
@@ -14,11 +14,7 @@ def destination
     private
 
     def submission_root_step
```

Comment:
Likewise here, we can put `edit('/steps/submission/more_information')` under `when :upload_finished` directly in the decision tree

Reply:
🤦‍♀️ corrected!